### ISSUE TITLE
Cache: HazelCast Update 4.0.3

### ISSUE THREAD

**Issue by melloware** [primefaces/primefaces#6271]
**Describe the defect**
Our build is currently failing because of this issue with HazelCast 4.0.1.

```
One or more dependencies were identified with known vulnerabilities in PrimeFaces:
hazelcast-4.0.1.jar/META-INF/maven/org.snakeyaml/snakeyaml-engine/pom.xml (pkg:maven/org.snakeyaml/snakeyaml-engine@1.0, cpe:2.3:a:snakeyaml_project:snakeyaml:1.0:*:*:*:*:*:*:*) : CVE-2017-18640
```

**Comment by melloware** [comment]
Looks like 4.0.2 is still vulnerable and there is an open issue to fix it for HazelCast 4.1.
https://github.com/hazelcast/hazelcast/issues/17425

**Comment by melloware** [comment]
https://github.com/hazelcast/hazelcast/issues/17444
